components/admin_console/permission_schemes_settings/permission_schemes_settings.jsx

```diff
@@ -2,24 +2,77 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
+import LoadingScreen from 'components/loading_screen.jsx';
+
+import PermissionsSchemeSummary from './permissions_scheme_summary';
+
+const PAGE_SIZE = 30;
+
 export default class PermissionSchemesSettings extends React.PureComponent {
-    renderTitle() {
-        return (
-            <FormattedMessage
-                id='admin.permissions.permissionSchemes'
-                defaultMessage='Permission Schemes'
-            />
-        );
+    static propTypes = {
+        schemes: PropTypes.array.isRequired,
+        actions: PropTypes.shape({
+            loadSchemes: PropTypes.func.isRequired,
+            loadSchemeTeams: PropTypes.func.isRequired,
+        }),
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+            loadingMore: false,
+            page: 0,
+        };
+    }
+
+    static defaultProps = {
+        schemes: [],
+    };
+
+    componentDidMount() {
+        this.props.actions.loadSchemes('team', 0, PAGE_SIZE).then((schemes) => {
+            var promises = [];
+            for (var scheme of schemes.data) {
+                promises.push(this.props.actions.loadSchemeTeams(scheme.id));
+            }
```

Comment:
Not requesting a change, but just a note that `scheme` will be available outside of the `{}` scope because it is declared with `var` vs `let`. I'm not aware of it making a difference in terms of GC (although hypothetically it might need to keep the variable around until the end of the function... but the JS engines may be smart enough to know that it isn't accessed and collect it sooner). Sounds like a fun experiment for me!

Reply:
Good point, normally I always use let instead of var, probably that code comes from some kind of copy/paste. I think is better to use let, at least in this case, so I going to change it.

Reply:
Done! :)